### DISCUSSION TITLE
How to compute changes to a row

### DISCUSSION THREAD

**Discussion by MrTin** [prisma/prisma#14177] [Q&A] [answered]
I've been looking through the APIs and documentation looking for a way to calculate the changes to a record after a successful commit but haven't come across this. Is there a recommended approach to do this with Prisma?

Coming from Rails there is [`ActiveModel::Dirty`](https://api.rubyonrails.org/classes/ActiveModel/Dirty.html), is there something equivalent in Prisma or should I just write a helper to compare the initial and changed state (easy enough but want to avoid reinventing the wheel)?

For example, let's say I have a model with 12 columns but I only update two of them -- I'm looking for a way to know that only these two keys have changed.

Thanks,
Martin

**Comment by ruheni** [comment, marked as answer]
Hey 👋🏽

Prisma currently doesn't support this pattern. I would encourage you to open a [feature request](https://github.com/prisma/prisma/issues/new/choose), and +1 it to help our Product and engineering teams prioritize the issue. 🙂

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
